resources/css/filament/admin/theme.css

```
@@ -1,4 +1,407 @@
+ * Filament source (vendor/.../modal.css):
+ *   - .fi-modal-window has `ring-1 ring-gray-950/5 dark:ring-white/10`
+ *   - .fi-modal-content has `py-6` (vertical) + `px-6` (horizontal via separate rules)
+ *   - .fi-modal-header has `px-6 pt-6`, footer `pb-6 px-6`
+ *
+ * !important needed to defeat utility hoisting. Scoped to admin panel; system
+ * panel installer wizard doesn't use modals.
+ * ============================================================================ */
+
+.fi-panel-admin .fi-modal-window {
+  --tw-ring-shadow: 0 0 #0000 !important;
+  box-shadow: var(--tw-shadow, 0 0 #0000) !important;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)